.npmrc

```diff
@@ -3,3 +3,7 @@ shared-workspace-lockfile=false
 
 # without this, installing one project installs all
 recursive-install=false
+
+# so client postinstall hook is not cached for all
+# https://github.com/pnpm/pnpm/releases/tag/v8.7.1
+side-effects-cache=false
```

Comment:
Does not actually fix the issue because we have dependency caching, but works if there's no caching.